lib/puppetserver/ca/action/delete.rb

```diff
@@ -192,6 +198,12 @@ def run(args)
             deleted_count += count
           end
 
+          if args['all']
+            certnames = Dir.glob("#{cadir}/signed/*.pem").map{ |c| File.basename(c, '.pem') }
+            # Since we don't run this with any other flags, we can set these variables directly
+            deleted_count, errored = delete_certs(cadir, certnames)
```

Comment:
Should error_on_not_found be set to false here? Its just walking the directory and deleting them, so I am thinking that error message may be misleading. What do you think?

Reply:
It theoretically shouldn't ever pass in a certname it can't find, since it's just picking up files on disk, so I don't think the error will ever actually show in the first place.